src/utils/utils.test.ts

```diff
@@ -0,0 +1,12 @@
+import { MAX_FILM_LENTH } from '../const';
+import { formatTime } from './utils';
+
+describe('Function: formatTime', () => {
+  it('should return "true" when output format is correct', () => {
```

Comment:
В названии теста нужно описывать не то что происходит в тесте, а кейс который в нём проверяется, например так:
```suggestion
  it('should return correct format when duration more than hour', () => {
```
И в тесте нужно будет вызвать `formatTime` с соответствующим количеством секунд, и проверить что результат получился с часами, минутами и секундами.